hw/top_darjeeling/data/top_darjeeling_seed.dev.hjson

```diff
@@ -0,0 +1,15 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+//////////////////////////////////////////////////////////////////////////////////////
+// Seed configuration for topgen and subsequent flows                               //
+// 256 bit seed for compile-time random constants                                   //
```

Comment:
nit: 256-bit seeds (there are multiple)

Reply:
Is it worth mentioning that the seeds must be different?

Reply:
Fixing that.